pom.xml

```diff
@@ -924,6 +924,11 @@
                 <artifactId>kapua-sso-api</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Please change header to: "    Copyright (c) 2016, 2020 Eurotech and/or its affiliates and others"